RubberduckTests/Inspections/SheetAccessedUsingStringInspectionTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Rubberduck.Inspections.Concrete;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.VBEditor.SafeComWrappers;
+using RubberduckTests.Mocks;
+
+namespace RubberduckTests.Inspections
+{
+    [TestFixture]
+    public class SheetAccessedUsingStringInspectionTests
+    {
+        [Test]
+        [Category("Inspections")]
+        public void SheetAccessedUsingString_ReturnsResult_AccessingUsingStringLiteral()
+        {
+            const string inputCode =
+                @"Public Sub Foo()
+    ThisWorkbook.Worksheets(""Sheet1"").Range(""A1"") = ""Foo""
+    ThisWorkbook.Sheets(""Sheet1"").Range(""A1"") = ""Foo""
+End Sub";
+
+            using (var state = ArrangeParserAndParse(inputCode))
+            {
+                var inspection = new SheetAccessedUsingStringInspection(state);
+                var inspectionResults = inspection.GetInspectionResults(CancellationToken.None);
+
+                Assert.AreEqual(2, inspectionResults.Count());
+            }
+        }
+
+        [Test]
+        [Category("Inspections")]
+        public void SheetAccessedUsingString_DoesNotReturnResult_AccessingUsingVariable()
+        {
+            const string inputCode =
+                @"Public Sub Foo()
+    Dim s As String
+    s = ""Sheet1""
+
+    ThisWorkbook.Worksheets(s).Range(""A1"") = ""Foo""
+    ThisWorkbook.Sheets(s).Range(""A1"") = ""Foo""
+End Sub";
+
+            using (var state = ArrangeParserAndParse(inputCode))
+            {
+                var inspection = new SheetAccessedUsingStringInspection(state);
+                var inspectionResults = inspection.GetInspectionResults(CancellationToken.None);
+
+                Assert.AreEqual(0, inspectionResults.Count());
+            }
+        }
+
+        [Test]
+        [Category("Inspections")]
+        public void InspectionName()
```

Comment:
This test can be removed, because there's [one test](https://github.com/rubberduck-vba/Rubberduck/blob/next/RubberduckTests/Inspections/GeneralInspectionTests.cs#L43) that uses reflection to get all inspections and validate this specific behavior.

Reply:
Test removed.